### PR TITLE
User backend

### PR DESCRIPTION
**Pull request by vincent-vu280**
# Description

- Fixed issue with the back end not recognizing data in JSON format

- Fixed delete not working

- Made the back end response data format so that it returns a single object rather than an array

### CODE REVIEW

File: server/controller/userController.js [hunk 1]
```diff
@@ -79,11 +79,11 @@ exports.deleteUser = async (req, res, next) => {
             });
         }
 
-        await User.remove({googleID : req.params.id});
+        await User.deleteOne({googleID : req.params.googleId});
 
         return res.status(200).json({
             success: true,
```
Comment (ReshmiCode) [inline server/controller/userController.js:85]: aren't we removing Sucess bc 200 itself it a success?

Comment (vincent-vu280) [reply server/controller/userController.js:85]: Have success gives more readability.

Comment (ReshmiCode) [reply server/controller/userController.js:85]: nO

Comment (SaloniSS) [reply server/controller/userController.js:85]: Good Programming Practices

Comment (ReshmiCode) [reply server/controller/userController.js:85]: then can we change data to be payload or something so it's not data.data

Comment (ReshmiCode) [reply server/controller/userController.js:85]: then can we change data to payload or something so it's not data.data

Comment (SaloniSS) [reply server/controller/userController.js:85]: Ig

**Comment by ReshmiCode** [comment]
@vincent-vu280 just switch `data: user` etc.. to every time you use data just make it `payload` instead

**Pull request merged**